wiki/List_of_Guides/Live_streaming_osu!/en.md

```diff
@@ -0,0 +1,317 @@
+# Live streaming osu!
+
+If you are visiting this page, it is likely you have an interest and want to learn how to live stream *osu!* on twitch.tv. Doing so is a very simple process but may require a reliable computer to be able to run both *osu!* and send live encoding to [twitch.tv](https://www.twitch.tv) (which from now on will be referred to as TTV) at the same time.
+
+## Step 1 - Registration
+
+![Registration Steps for Twitch.tv](img/TTV_RegistrationSteps.jpg "Registration Steps for Twitch.tv")
+
+1.  Registering for an account for TTV is very easy, just head to the website [twitch.tv](https://www.twitch.tv) and click on the `Sign Up` button in the top right corner.
+2.  Afterwards, fill in the fields.
+3.  You will get an account activation link via e-mail.
+4.  Once you are done, your channel will be at `www.twitch.tv/{username}` (username being your chosen username).
+
+## Step 2 - Prepare twitch.tv for streaming
+
+Twitch.tv is a gaming community, and as such, there will be *a lot* of gaming libraries to select from. However, you only need to worry about finding *osu!* since that is what this guide is about.
+![Channel Setup](img/TTV_ChannelSetup.jpg "Channel Setup")
+
+1.  Once you have logged in, click on your username at the top right corner.
+2.  Click on `Dashboard` from the drop-down list.
+3.  Input the title and click on the icon where it says `Not Playing` to change into `Playing` then type "osu!". A list of games with similar names should appear, the one you want is *osu!*
+
+Once you are done with that, you are ready to start streaming if you want. There is one last question remaining, how can you make TTV recognize your *osu!* window and live stream it? We will cover that in the next step. If you already have a preferred program you want use for this then go ahead and start streaming, otherwise read on.
+
+**Warning**: If you do not pause/mute the stream preview in either your channel or dashboard, your viewers may get echo. Also, you may click on the wheel icon on the chat and click on `popout` to pop out the chatroom and put it wherever you want without the need to constantly have your browser open.
+
+## Step 3 - Live streaming programs
+
+### XSplit
+
+#### Introduction
+
+[XSplit](https://www.xsplit.com/) is one of the most common live streaming programs nowadays, specially because it does not cause too much stress on your computer and comes with a neat screen-capture feature. The downside? Some options are disabled and have lower quality unless you buy the license.
```

Comment:
`The downside? Some options are disabled and have lower quality unless you buy the license.`
Delete because of `Cons`